commcare-cloud/commcare_cloud.py

```diff
@@ -165,37 +172,52 @@ def get_ansible_vault_password():
         exit(exit_code)
 
 
-class UpdateConfig(object):
+class _AnsiblePlaybookAlias(object):
+    @staticmethod
+    def make_parser(parser):
+        arg_skip_check(parser)
+        arg_branch(parser)
+
+
+class UpdateConfig(_AnsiblePlaybookAlias):
     command = 'update-config'
     help = (
         "Run the ansible playbook for updating app config "
         "such as django localsettings.py and formplayer application.properties."
     )
 
-    @staticmethod
-    def make_parser(parser):
-        arg_skip_check(parser)
-        arg_branch(parser)
-
     @staticmethod
     def run(args, unknown_args):
         args.playbook = 'deploy_localsettings.yml'
         unknown_args += ('--tags=localsettings',)
         AnsiblePlaybook.run(args, unknown_args)
 
 
-class BootstrapUsers(object):
+class RestartElasticsearch(_AnsiblePlaybookAlias):
+    command = 'restart-elasticsearch'
+    help = (
+        "Do a rolling restart of elasticsearch."
+    )
+
+    @staticmethod
+    def run(args, unknown_args):
+        args.playbook = 'es_rolling_restart.yml'
+        if not ask_YES('Have you stopped all the elastic pillows?'):
+            exit(0)
+        puts(colored.yellow("For ES clusters without data redundancy, "
```

Comment:
I am pretty sure this is all clusters now. I believe even on clusters with some replica shards its only on a couple indices, so this might want to be worded more strongly.